sktime/datatypes/_hierarchical/_check.py

```diff
@@ -78,7 +78,7 @@ def _ret(valid, msg, metadata, return_metadata):
         return valid
 
 
-def check_pdmultiindex_hierarchical(obj, return_metadata=False, var_name="obj"):
+def check_pdmultiindex_Hierarchical(obj, return_metadata=False, var_name="obj"):
```

Comment:
note to self: this should be lower_snake_case, should be reverted. Probably a merge accident.

Reply:
fixed